EIPS/eip-3540.md

```diff
@@ -253,32 +261,168 @@ All cases should be checked for creation transaction, `CREATE` and `CREATE2`.
 - `JUMP`/`JUMPI` to `5B` (`JUMPDEST`) byte outside of code section - exceptional abort
 - EOF code containing `PC` opcode - offset inside code section is returned
 - `PUSH*` instructions
-    - Complete push data - no changes expected
-    - Truncated push data without data section - execution ends with exceptional abort
-    - Truncated push data with data section - execution ends with exceptional abort
+  - Complete push data - no changes expected
+  - Truncated push data without data section - execution ends with exceptional abort
+  - Truncated push data with data section - execution ends with exceptional abort
 - Execution flows out of code section bounds (i.e. PC gets to `code_section_size`) - exceptional abort
 - EOF code containing `CODECOPY/CODESIZE` - works as in legacy code
-    - `CODESIZE` returns the size of entire container
-    - `CODECOPY` can copy from code section
-    - `CODECOPY` can copy from data section
-    - `CODECOPY` can copy from the EOF header
-    - `CODECOPY` can copy entire container
+  - `CODESIZE` returns the size of entire container
+  - `CODECOPY` can copy from code section
+  - `CODECOPY` can copy from data section
+  - `CODECOPY` can copy from the EOF header
+  - `CODECOPY` can copy entire container
 - `EXTCODECOPY/EXTCODESIZE/EXTCODEHASH` with the EOF *target* contract - works as with legacy target contract
-    - `EXTCODESIZE` returns the size of entire target container
-    - `EXTCODEHASH` returns the hash of entire target container
-    - `EXTCODECOPY` can copy from target's code section
-    - `EXTCODECOPY` can copy from target's data section
-    - `EXTCODECOPY` can copy from target's EOF header
-    - `EXTCODECOPY` can copy entire target container
-    - Results don't differ when executed inside legacy or EOF contract
+  - `EXTCODESIZE` returns the size of entire target container
+  - `EXTCODEHASH` returns the hash of entire target container
+  - `EXTCODECOPY` can copy from target's code section
+  - `EXTCODECOPY` can copy from target's data section
+  - `EXTCODECOPY` can copy from target's EOF header
+  - `EXTCODECOPY` can copy entire target container
+  - Results don't differ when executed inside legacy or EOF contract
 
 ## Reference Implementation
 
 ### Generic Implementation
 
 ```python
 FORMAT = 0xEF
-MAGIC = 0x00 # To be defined
+MAGIC = 0x00
```

Comment:
These two versions of the code that are being added are in fact outdated, should be removed.